doc/tutorials/plugins.md

```diff
@@ -11,45 +11,45 @@ and what various methods exists within one.
 
 ### The Interface ###
 
-All plug-ins use the same basic interface. This interface consists of five basic functions, 
```

Comment:
I think you can do `rebase -i` and just drop the commit. (Warning: its a potentially dangerous operation: you might lose data)

If you cannot do it: I can still accept the PR, just split the PR (in such situations) in the future.